prow/git/git.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package git provides a client to plugins that can do git operations.
+package git
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sync"
+)
+
+const github = "https://github.com"
+
+// Client can clone repos. It keeps a local cache, so successive clones of the
+// same repo should be quick. Create with NewClient. Be sure to clean it up.
+type Client struct {
+	dir  string
+	git  string
```

Comment:
Seems like the path to the git binary but it's not obvious. Can you add a comment about it?